lakefile.lean

```diff
@@ -11,3 +11,6 @@ lean_lib Mathlib
 lean_exe runLinter where
   root := `scripts.runLinter
   supportInterpreter := true
+
+meta if get_config? env = some "dev" then -- dev is so not everyone has to build it
```

Comment:
```suggestion
meta if get_config? doc = some "on" then -- do not download and build doc-gen4 by default
```
The `env` option doesn't mesh well with `lake update`, at least if we ever add a second env.  For `lake update`, all dependencies need to be present.  This isn't possible with `lake -Kenv=dev -Kenv=test update`, but only with separate options `lake -Kdoc=on -Ktest=on update`.